test/blackbox-tests/test-cases/dune-cache/cache-man.t

```diff
@@ -5,39 +5,18 @@ Here we observe the documentation for the dune cache commands.
          dune-cache - Manage the shared cache of build artifacts
   
   SYNOPSIS
-         dune cache [--size=BYTES] [--trimmed-size=BYTES] [OPTION]… [ACTION]
+         dune cache COMMAND …
   
   DESCRIPTION
          Dune can share build artifacts between workspaces. Currently, the only
          action supported by this command is `trim`, but we plan to provide
-         more functionality soon. 
+         more functionality soon.
   
-  ACTIONS
-         trim trim the shared cache to free space.
-  
-  ARGUMENTS
-         ACTION
-             The cache action to perform (trim)
-  
-  OPTIONS
-         --size=BYTES
-             Size to trim the cache to.
```

Comment:
Where did this text go? I don't think it's anywhere to be found in the new version of the test. Do we just not print it anymore? That seems wrong.

Reply:
Oops, found it now, thanks!